src/main/java/com/conveyal/datatools/common/utils/SparkUtils.java

```diff
@@ -234,6 +234,8 @@ public static void logRequestOrResponse(
             LOG.warn("Request object is null. Cannot log.");
             return;
         }
+        // don't log job status requests/responses, they clutter things up
+        if (request.pathInfo().contains("status/jobs")) return;
```

Comment:
Lol, I feel like I've been advocating for this for a long time... what made you change your mind?